internal/api/index.go

```diff
@@ -16,6 +16,29 @@ import (
 	"github.com/photoprism/photoprism/pkg/txt"
 )
 
+// GET /api/v1/index
+func IndexingStatus(router *gin.RouterGroup) {
+	router.GET("/index", func(c *gin.Context) {
+		s := Auth(SessionID(c), acl.ResourcePhotos, acl.ActionRead)
```

Comment:
I'd use `ActionUpdate` as guests, who can read photos, probably don't need access to this.